modules/dnn/CMakeLists.txt

```diff
@@ -21,7 +21,15 @@ if(OPENCV_DNN_OPENCL AND HAVE_OPENCL)
   add_definitions(-DCV_OCL4DNN=1)
 endif()
 
-ocv_option(OPENCV_DNN_CUDA "Build with CUDA support" HAVE_CUDA AND HAVE_CUBLAS AND HAVE_CUDNN)
+if(NOT DEFINED OPENCV_DNN_CUDA AND HAVE_CUDNN AND CUDNN_VERSION VERSION_LESS 8.0)
+  message(STATUS "DNN: CUDNN 8.0 is not supported yes. Details: https://github.com/opencv/opencv/issues/17496")
```

Comment:
Yet not yes. There is a typo :)

Reply:
Hopefully this message will be removed soon =)

Reply:
Thanks for this , still i am getting error 
"OpenCV/modules/cudacodec/src/precomp.hpp:59:14: fatal error: nvcuvid.h: No such file or directory
   59 |     #include <nvcuvid.h>"